### ISSUE TITLE
[Bug] v1.0.3* conda installation error VEBA-assembly_env  perl-xml-parser-2.44-4 

### ISSUE THREAD

**Issue by erfanshekarriz** [jolespin/veba#15]
Dear Josh, 

I'm setting up the new version of VEBA 1.03d on a colleague's computer and there seems to be a conflict when setting up the conda environment.

I'm specifically getting this error:
```
Could not solve for environment specs
Encountered problems while solving:
  - package perl-xml-parser-2.44-4 requires perl >=5.22.0,<5.23.0, but none of the providers can be installed

The environment can't be solved, aborting the operation 
```

perl-xml-parser-2.44-4 seems to need an extremely narrow range of perl versions that it can work with. Any workaways around this?

Best, 

Erfan

**Comment by erfanshekarriz** [comment]
I've just checked the commits history and saw that you previously had 
`perl=5.26.2=h36c2ea0_1008` in place of ```perl=5.32.1=2_h7f98852_perl5```. 

I think this is causing some problems. Let me know if you find any solution. Until then I can download a previous version. 

Best, 

Erfan

**Comment by jolespin** [comment]
I was experiencing this issue earlier last week but thought I resolved it. I’ll get this resolved in a few hours and get back to you. 

Apologies for the inconvenience.

**Comment by jolespin** [comment]
So i've made some edits since v1.0.3d (need to release a v1.0.3e soon). 

I just downloaded the repository and installed the assembly environment.  Can you try from the current repo? 

```
(base) git clone https://github.com/jolespin/veba
Cloning into 'veba'...
remote: Enumerating objects: 985, done.
remote: Counting objects: 100% (65/65), done.
remote: Compressing objects: 100% (47/47), done.
remote: Total 985 (delta 34), reused 43 (delta 18), pack-reused 920
Receiving objects: 100% (985/985), 2.55 MiB | 10.13 MiB/s, done.
Resolving deltas: 100% (639/639), done.
(base) conda env create -n test_env -f veba/install/environments/VEBA-
VEBA-amplicon_env.yml             VEBA-binning-eukaryotic_env.yml   VEBA-classify_env.yml             VEBA-mapping_env.yml
VEBA-annotate_env.yml             VEBA-binning-prokaryotic_env.yml  VEBA-cluster_env.yml              VEBA-phylogeny_env.yml
VEBA-assembly_env.yml             VEBA-binning-viral_env.yml        VEBA-database_env.yml             VEBA-preprocess_env.yml
(base) conda env create -n test_env -f veba/install/environments/VEBA-assembly_env.yml
Collecting package metadata (repodata.json): done
Solving environment: done


==> WARNING: A newer version of conda exists. <==
  current version: 22.9.0
  latest version: 22.11.1

Please update conda by running

    $ conda update -n base -c defaults conda


Preparing transaction: done
Verifying transaction: done
Executing transaction: done
#
# To activate this environment, use
#
#     $ conda activate test_env
#
# To deactivate an active environment, use
#
#     $ conda deactivate

Retrieving notices: ...working... done
```

**Comment by erfanshekarriz** [comment]
Hi Josh, 

Awesome and thanks for your help again. I tested out the developmental version and it's all fixed. 

Let me try and install all environments and get back to you in an hour if I run into any problems. 

Best, 

Erfan

**Comment by jolespin** [comment]
Fixed in the newest release (v1.0.3e): https://github.com/jolespin/veba/releases/tag/v1.0.3e
